docs-ref-conceptual/azure-cli-vm-tutorial-1.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Create virtual machine (VM) on a virtual network (VNet) prerequisites – Azure CLI | Microsoft Docs
 description: Prerequisites for creating a virtual machines (VM) on a virtual network (VNet) with the Azure CLI.
-ms.date: 08/2/2023
+ms.date: 01/08/2024
```

Comment:
@v-abdriaz , Please remember to update the `ms.date` when you refresh an article.  I have completed this in all 5 files within your branch.  Just remember for next time.